src/announcement-data/systems/rolling-stock/WMTClass172.ts

```diff
@@ -66,6 +66,10 @@ export default class WMTClass172 extends TrainAnnouncementSystem {
   readonly FILE_PREFIX = 'WMT/172'
   readonly SYSTEM_TYPE = 'train'
 
+  private readonly StationsWithForcedChangeHere = {
+    SBJ: 'stations.SBT',
```

Comment:
Is it worth making the "change here" stations an array instead? It would make it easier if there are other locations which play similar messages with more than one station is announced.

Up to you; happy to merge anyway.